.github/workflows/pull-request.yml

```
@@ -2,7 +2,7 @@ name: "Test pull requests"
 
 on:
   workflow_dispatch:  # manual trigger for debugging
-  pull_request:
+  # pull_request:
```

Maybe add a comment:
```suggestion
  # pull_request:     # TODO/FIXME: enable this again once everything works
```
or so... alternatively open an issue to remind us